ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/TheirDetailsMapper.java

```diff
@@ -100,10 +101,21 @@ private TheirDetails getCompany(CCDParty ccdParty) {
 
     private TheirDetails getIndividual(CCDParty ccdParty) {
         CCDIndividual ccdIndividual = ccdParty.getIndividual();
-        return new IndividualDetails(ccdIndividual.getName(), addressMapper.from(ccdIndividual.getAddress()),
-            ccdIndividual.getEmail(), representativeMapper.from(ccdIndividual.getRepresentative()),
+        return new IndividualDetails(ccdIndividual.getName(),
+            addressMapper.from(ccdIndividual.getAddress()),
+            ccdIndividual.getEmail(),
+            representativeMapper.from(ccdIndividual.getRepresentative()),
             addressMapper.from(ccdIndividual.getCorrespondenceAddress()),
-            LocalDate.parse(ccdIndividual.getDateOfBirth(), ISO_DATE));
+            parseDob(ccdIndividual.getDateOfBirth())
+        );
+    }
+
+    private LocalDate parseDob(String dateOfBirth) {
+        if (StringUtils.isBlank(dateOfBirth)) {
+            return null;
```

Comment:
I think date of birth is mandatory for `Individual`. Is there a scenario where it would be blank here?

Reply:
it is no mandatory for solicitor

Reply:
mandatory or not mandatory?

Reply:
no dob for solicitor

Reply:
dateofbirth isn't mandatory for their details anyway